src/main/cpp/hw/DragonSparkMax.cpp

```diff
@@ -200,25 +181,27 @@ void DragonSparkMax::SetVoltage(units::volt_t output)
 
 bool DragonSparkMax::IsMotorInverted() const
 {
+    return m_spark->GetInverted();
 }
 
 bool DragonSparkMax::IsForwardLimitSwitchClosed()
 {
-    return false;
+    return m_spark->GetForwardLimitSwitch(m_forwardType).Get();
 }
 
 bool DragonSparkMax::IsReverseLimitSwitchClosed()
 {
-    return false;
+    return m_spark->GetReverseLimitSwitch(m_reverseType).Get();
```

Comment:
This may have the same problem as GetEncoder() where GetForwardLimitSwitch() cannot be called more than once and we need to store the value.